variable/technology/electricity grid.yaml

```diff
@@ -0,0 +1,25 @@
+# List of variables related to the electricity grid
+
+Frequency Containment Reserve Demand|Electricity|Grid:
```

Comment:
This is a very long top-level category name. Not sure what the best approach here is - what do you think of `Grid|Electricity|Demand|Reserve|Frequency Containment` (or may be `Network` for generality?). This will mean that all related variables will be shown next to each other in the Scenario Explorer (where the variable selection is organised in a tree structure).

This suggestion also applies to the other variables proposed here...

Reply:
Agree, done